babel.config.js

```diff
@@ -1 +1,3 @@
-module.exports = { babelrcRoots: ['./api-server', './client'] };
+module.exports = {
+  babelrcRoots: ['./api-server', './client', 'tools/ui-components']
+};
```

Comment:
We are using rollup to bundle the contents of the ui-components directory and it is isolated from the babel config from the root directory.

Reply:
Makes sense, but in `package.json` the script `test-ui-components` refers to jest from project root; which requires babel to run the tests successfully. I mean tests fail if we remove config from here. Any suggestion?

Reply:
Thanks @ahmadabdolsaheb  I am able to resolve these as well 🌟